backend/src/Tabulators/Approval.test.ts

```diff
@@ -0,0 +1,97 @@
+import { Approval } from './Approval'
+
+describe("Approval Tests", () => {
+    test("Single Winner Test", () => {
+        // Simple test to elect the candidate with most votes
+        const candidates = ['Alice', 'Bob', 'Carol', 'Dave']
+
+        const votes = [
+            [1, 1, 1, 1],
```

Comment:
At the moment only ``0, 0, 0, 0`` is considered an under vote, but ``1, 1, 1, 1`` has the same effect on the election results, so we could also consider classifying that as an undervote